myth/Api/Server/ApiController.php

```diff
@@ -186,41 +185,39 @@ public function __construct()
 			unset($file);
 		}
 
-	    parent::__construct();
-
 		$this->config->load('api');
 
 		// Gather config defaults when a value isn't set for this controller
-		if ( empty($this->enable_logging) ) $this->enable_logging = config_item('api.enable_logging');
-		if ( empty($this->enable_rate_limits) ) $this->enable_rate_limits = config_item('api.enable_rate_limits');
-		if ( empty($this->rate_limits) ) $this->rate_limits = config_item('api.rate_limits');
+		if (empty($this->enable_logging)) $this->enable_logging = config_item('api.enable_logging');
+		if (empty($this->enable_rate_limits)) $this->enable_rate_limits = config_item('api.enable_rate_limits');
+		if (empty($this->rate_limits)) $this->rate_limits = config_item('api.rate_limits');
 
 		// Should we restrict to SSL requests?
 		if (config_item('require_ssl') === true && ! $this->request->ssl)
 		{
-			$this->failForbidden( lang('api.ssl_required') );
```

Comment:
I have a feeling it's a setting in your IDE, but why are you getting rid of all of my spaces? I typically only leave them around parameters when the parameter is a function and it makes it faster to read with the extra space so you can clearly see where the other function ends.